src/math/ceil.js

```diff
@@ -0,0 +1,35 @@
+const ceil = (number, precision) => {
+    let numberString = number.toString();
+    let decimalIndex = numberString.indexOf('.');
+    let beforeDecimal = numberString.substring(0, decimalIndex);
+    let afterDecimal = numberString.substring(decimalIndex, numberString.length);
+  if (Number.isInteger(number)){
+    return number;
+  } else if (Number.isInteger(number) === false) {
+      if (precision === undefined || precision  === 0) {
+        for(let i = 0; i < afterDecimal.length; i++){
+          if(numberString[i] > 0){
+            let roundUp = parseInt(beforeDecimal) + 1;
+            return roundUp;
+          }
+        }
+      } else if (precision > 0) {
+          let beforeDecimalArray = [];
+          beforeDecimalArray.push(beforeDecimal);
+          let afterDecimalArray = afterDecimal.split('');
+          for(let j = 1; j < afterDecimalArray.length; j++){
+            if(j === precision){
+              afterDecimalArray.splice(j, 1, parseInt(afterDecimalArray[j])+1);
+              afterDecimalArray.splice(j, 1, afterDecimalArray[j].toString());
+            }
+          }
+          afterDecimalArray.unshift(beforeDecimalArray[0]);
```

Comment:
Unshifting was necessary to cover a float situation.  We initially separated the number before decimal and after decimal in order to manipulate them separately. Here we are putting them back together after rounding up the numbers following the decimal.